web/src/views/SubmissionPortal/Components/TitleBanner.vue

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+import {
+  defineComponent,
+} from '@vue/composition-api';
+
+export default defineComponent({
+  components: {},
+  setup() {
+    return {};
+  },
+});
+</script>
+
+<template>
+  <v-container>
+    <v-row :style="{ backgroundImage: 'url(' + require('@/assets/submission-portal-hero.png') + ')', backgroundSize: 'cover' }">
```

Comment:
Hooray! Setting an image to background image when it is not info bearing is excellent accessibility practice. :)